src/modules/Accordion/Accordion.js

```diff
@@ -22,7 +22,10 @@ export default class Accordion extends AutoControlledComponent {
     activeIndex: PropTypes.number,
 
     /** Accordion.Title and Accordion.Content components.  Mutually exclusive with the panels prop. */
-    children: customPropTypes.mutuallyExclusive('panels'),
+    children: customPropTypes.all([
+      customPropTypes.mutuallyExclusive(['panels']),
+      PropTypes.node,
+    ]),
```

Comment:
Here is how we use the `all()` validator.  It takes an array of prop type functions.